packages/react-vapor/src/components/radio/examples/RadioExamples.tsx

```diff
@@ -6,6 +6,8 @@ import {SingleRadioExample} from './SingleRadioExample';
 import {SingleRadioWithDescriptionExample} from './SingleRadioWithDescriptionExample';
 
 export class RadioExamples extends React.Component<any, any> {
+    static description =
+        'Radio Buttons allow the selection of only one option inside a set (e.g. when you need to ask the gender or marital status).';

```

Comment:
```suggestion
        'Radio Buttons allow the selection of only one option inside a set (e.g. when you need to ask the marital status).';
```
I'd remove the genre example to be more politically correct in 2019.

Reply:
I removed the examples all together. They don't really make sense in the description anyway